Funcky.Money/Visitors/ToHumanReadableVisitor.cs

```diff
@@ -0,0 +1,48 @@
+using System.Collections.Generic;
+
+namespace Funcky
+{
+    internal class ToHumanReadableVisitor : IMoneyExpressionVisitor
+    {
+        private readonly Stack<string> _stack = new();
+
+        public string Result
+            => _stack.Peek();
+
+        public void Visit(Money money)
+        {
+            _stack.Push(string.Format($"{{0:N{money.Currency.MinorUnitDigits}}}{{1}}", money.Amount, money.Currency.AlphabeticCurrencyCode));
```

Comment:
Why doesn't this use the same formatting as `Money.ToString`?

Reply:
Because I want a representation which is unambigious, ideally parsable at a later point without explicit knowledge which currencies are involved.

$2.00 is ambigious while 2.00USD is not.

Reply:
Ah makes sense